src/eradiate/scenes/measure/_core.py

```diff
@@ -69,10 +71,13 @@
 
 
 def _measure_spectral_config_srf_converter(value: t.Any) -> Spectrum:
-    if isinstance(value, str):
-        with data.open_dataset(f"spectra/srf/{value}.nc") as ds:
-            w = ureg.Quantity(ds.w.values, ds.w.attrs["units"])
-            srf = ds.data_vars["srf"].values
+    if isinstance(value, (str, os.PathLike)):
+        if str(value).endswith(".nc"):  # path to local file
+            ds = xr.load_dataset(value)
+        else:  # keyword for file in data store
+            ds = data.load_dataset(f"spectra/srf/{value}.nc")
```

Comment:
This won't allow loading custom files from the data store, which IMO is useful for testing purposes. Following the logic of `SolarIrradianceSpectrum`, I suggest something a bit more flexible:

```suggestion
        ds = converters.load_dataset( 
            f"spectra/srf/{value}.nc" 
            if not str(value).endswith(".nc")  # keyword for file in data store
            else value  # path to file (local or in data store)
        )
```